security/nodeagent/templates/daemonset.yaml

```diff
@@ -1,47 +1,52 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: istio-nodeagent
+  name: istio-sdsagent
   namespace: {{ .Release.Namespace }}
   labels:
-    app: istio-nodeagent
-    istio: nodeagent
+    app: istio-sdsagent
+    istio: sdsagent
     release: {{ .Release.Name }}
 spec:
   selector:
     matchLabels:
-      istio: nodeagent
+      istio: sdsagent
   template:
     metadata:
       labels:
-        app: istio-nodeagent
-        istio: nodeagent
+        app: istio-sdsagent
+        istio: sdsagent
         release: {{ .Release.Name }}
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
-      serviceAccountName: istio-nodeagent-service-account
+      serviceAccountName: istio-sdsagent
```

Comment:
whats the reason for this change? I think all other components have `-service-account`

Reply:
For no reason. It's a bit tricky to change for old components ( pilot in particular ), but at least we can stop doing it.